docs/source/simulator_support.rst

```diff
@@ -79,16 +73,17 @@ In order to use this simulator, set :make:var:`SIM` to ``verilator``:
     A working installation of `Verilator <https://www.veripool.org/verilator/>`_ is required.
     You can find installation instructions `in the Verilator documentation <https://verilator.org/guide/latest/install.html>`_.
 
-.. note::
-
-    Delayed assignments require the use of Verilator's `timing <https://verilator.org/guide/latest/exe_verilator.html#cmdoption-timing>`_ argument.
-
-To run cocotb with Verilator, you need ``verilator`` in your PATH.
-
 .. versionadded:: 1.3
 
 .. versionchanged:: 1.5 Improved cocotb support and greatly improved performance when using a higher time precision.
 
+.. versionchanged:: 2.0
+
+    Reimplemented the Verilator evaluator loop used in cocotb tests.
+    This allowed for better performance and behavior more consistent with event-based simulators.
+    Additionally, added support for inertial writes so :envvar:`COCOTB_TRUST_INERTIAL_WRITES` can be defaulted to ``1``,
+    which should noticeably improve performance.
```

Comment:
```suggestion
    Additionally, added support for inertial writes,
    which noticeably improves performance.
```

On first reading I wasn't sure if the user can now set the environment variable, or if we set it by default to 1.